test/server/workers/testworkers.cpp

```diff
@@ -198,8 +237,21 @@ void TestWorkers::testCreatePlaceholder() {
                        ioError == IoError::Success);
 
         exitInfo = _syncPal->_executorWorker->createPlaceholder(relativeFilePath);
+#ifdef __APPLE__
         CPPUNIT_ASSERT_EQUAL(ExitCode::SystemError, exitInfo.code());
         CPPUNIT_ASSERT_EQUAL(ExitCause::FileAccessError, exitInfo.cause());
+#else
+        // Strangely (bug?), the Windows api is able to create a placeholder in a folder for which the user does not have rights
+        CPPUNIT_ASSERT_EQUAL(ExitCode::SystemError, exitInfo.code());
+        CPPUNIT_ASSERT_EQUAL(ExitCause::Unknown, exitInfo.cause());
+
+        // Remove placeholder
+        std::error_code ec;
+        std::filesystem::remove(_syncPal->localPath() / relativeFilePath);
+        if (ec) {
+            // Cannot remove file
+        }
```

Comment:
`ec` is not used at all. Remove it or use it.

Reply:
assert added